### PR TITLE
Home

### PR DESCRIPTION
**Pull request by plaso**

### CODE REVIEW

**Review by Cdelatorre — CHANGES_REQUESTED** [review]
very bad Pablo

File: app.js [hunk 1]
```diff
@@ -1 +1,3 @@
-console.log('hello world')
\ No newline at end of file
+console.log('hello world')
```
Comment (plaso) [inline app.js:1]: This is unnecessary

File: app.js [hunk 2]
```diff
@@ -1 +1,3 @@
...
+
```
Comment (Cdelatorre) [inline app.js:2]: remove console.logs()

**Review by Cdelatorre — APPROVED** [review]
Marcos is handsome

**Pull request merged**